flow/Platform.actor.cpp

```diff
@@ -4005,7 +4033,7 @@ TEST_CASE("/flow/Platform/getMemoryInfo") {
 	};
 
 	std::stringstream memInfoStream(memString);
-	getMemoryInfo(request, memInfoStream);
+	linux_os::getMemoryInfo(request, memInfoStream);
```

Comment:
This seems to be guarded by `#if defined(__linux__)`?

Reply:
The test itself is already wrapped by `#ifdef __linux__` guard.